package.json

```diff
@@ -48,8 +48,8 @@
         "loaders.css": "^0.1.2",
         "mocha": "^8.2.1",
         "prop-types": "^15.7.2",
-        "react": "^17.0.1",
-        "react-dom": "^17.0.1",
+        "react": "<17.0.0",

```

Comment:
Try to find the semver expression that gives you the latest version inside 16. The difference between this and that is that `10.0.0` would be a valid version given your expression.

Reply:
Done, used this https://semver.npmjs.com/